community/kernel/src/main/java/org/neo4j/kernel/impl/storemigration/StoreMigrator.java

```diff
@@ -330,11 +327,17 @@ private void rebuildCountsFromScratch( File storeDir, long lastTxId, PageCache p
         try ( NeoStores neoStores = storeFactory.openNeoStores( false ) )
         {
             NodeStore nodeStore = neoStores.getNodeStore();
+            nodeStore.makeStoreOk();
```

Comment:
You should not call `makeStoreOk` on individual stores anymore, always go through `NeoStores`... so first get all the stores you need, then call `NeoStores#makeStoreOk()`